sourcery_analytics/extractors.py

```diff
@@ -73,7 +76,7 @@ def extract(
     elif condition:
         extractor = Extractor[T].from_condition(condition)
     elif function:
-        extractor = Extractor.from_function(function)
+        extractor = Extractor[T].from_function(function)
```

Comment:
**suggestion (docstrings):** Please update the docstring for function: `extract`

Reason for update: Change in how Extractor.from_function is called requires an update.

Suggested new docstring:
```
Extract nodes based on the provided parameters. The extraction can be based on a specific type, condition, or a custom function.
```